Sources/SwiftTreeSitter/Node.swift

```diff
@@ -82,16 +82,55 @@ extension Node {
         return TSRange(points: pointRange, bytes: byteRange)
     }
 
+    public var isNull: Bool {
+        return ts_node_is_null(internalNode)
+    }
+
+    public var isExtra: Bool {
+        return ts_node_is_extra(internalNode)
+    }
+
+    public var isNamed: Bool {
+        return ts_node_is_named(internalNode)
+    }
+
+    public var isMissing: Bool {
+        return ts_node_is_missing(internalNode)
+    }
+
+    public var hasChanges: Bool {
+        return ts_node_has_changes(internalNode)
+    }
+
+    public var hasError: Bool {
+        return ts_node_has_error(internalNode)
+    }
+
+    public func childByFieldName(fieldName: String) -> Node? {
+        let n = ts_node_child_by_field_name(internalNode, fieldName, UInt32(fieldName.count))
```

Comment:
I missed this at first, but the function `ts_node_child_by_field_name` takes a C string and C string length. I'm actually surprised this even compiles! A `String` can be implicitly converted to `UnsafePointer<CChar>` ? Wow!

Regardless, I think we should probably do something more explicit like https://github.com/ChimeHQ/SwiftTreeSitter/blob/main/Sources/SwiftTreeSitter/Language.swift#L39

Reply:
I did a little more investigation here, it does indeed do an implicit conversion to UTF8, which is pretty cool. So, this can be used. Just need to change the character count to be explicitly UTF8. (Though I admit that in practice this will probably always be ASCII)

Reply:
Sure. I have updated the count generation based on your suggestion.